### PR TITLE
Removing not-used rating buttons.

### LINKED ISSUE #116: Fix button with stars
screen shot 2015-08-04 at 18 08 42
On hovering over first area one star should be active, on second 5 stars should be active

### PR DESCRIPTION
**Pull request by burakukula**
Fixes #116

### CODE REVIEW

File: src/components/buttons/buttons.html [hunk 1]
```diff
@@ -59,7 +59,7 @@ <h3 class="docs-block__header">Secondary buttons</h3>
     <div class="docs-block__content docs-block__content--align-top">
         <a class="sg-button-secondary">
             <div class="sg-button-secondary__hole">
-              <svg class="sg-icon sg-icon--x14 sg-icon--adaptive">
+                <svg class="sg-icon sg-icon--x14 sg-icon--adaptive">
```
Comment (aju) [inline src/components/buttons/buttons.html]: who broke the spaccccing?:O

File: src/components/buttons/buttons.html [hunk 2]
```diff
@@ -416,9 +350,8 @@ <h3 class="docs-block__header">Secondary button wide</h3>
...
         <button class="sg-button-secondary sg-button-secondary--full">This button is wide</button>
 
-        <button
-            class="sg-button-secondary sg-button-secondary--alt-inverse sg-button-secondary--small sg-button-secondary--full">
+        <button class="sg-button-secondary sg-button-secondary--alt-inverse sg-button-secondary--small sg-button-secondary--full">
             This one to!
         </button>
     </div>
-</section>
+</section>
```
Comment (aju) [inline src/components/buttons/buttons.html]: new line?:O

File: src/components/buttons/buttons.html (hunk 1 above)
Comment (kdzwinel) [inline src/components/buttons/buttons.html]: I'm so disappointed 😞

File: src/components/buttons/buttons.html (hunk 2 above)
Comment (kdzwinel) [inline src/components/buttons/buttons.html]: please add an empty line a the end of the file ( I think we have this defined in editorconfig )

File: src/components/buttons/buttons.html (hunk 1 above)
Comment (burakukula) [inline src/components/buttons/buttons.html]: @kdzwinel I am so sorry - I am hopeless.

**Comment by burakukula** [comment]
@aju @kdzwinel please have a look.

**Pull request merged**